integration-libs/opf/root/facade/opf-checkout.facade.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { facadeFactory, QueryState } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { OPF_CORE_FEATURE } from '../feature-name';
+import { ActiveConfiguration } from '../model/opf-checkout.model';
+
+@Injectable({
+  providedIn: 'root',
+  useFactory: () =>
+    facadeFactory({
+      facade: OpfCheckoutFacade,
+      feature: OPF_CORE_FEATURE,
+      methods: ['getActiveConfigurationsState', 'getActiveConfigurations'],
+    }),
+})
+export abstract class OpfCheckoutFacade {
+  /**
+   * Get checkout payment active configurations
+   */
+  abstract getActiveConfigurations(): Observable<ActiveConfiguration[]>;
+  /**
+   * Get checkout payment active configurations
+   */
+  abstract getActiveConfigurationsState(): Observable<
```

Comment:
Double-check if this method is needed and if it should be part of the public API. It is used only in the implementation of this facade inside `getActiveConfigurations()` to get data from the state.

Reply:
Actually, as times we might want to use facade instead of core method. I see this pattern in other places as well. Would leave it.

Reply:
If so, please change JSDoc, because both methods have the same description.

Reply:
Fixed. Removed explicit method as it was redundant.